backend/src/main/java/com/dbms/georgia_express/model/Card.java

```diff
@@ -0,0 +1,172 @@
+package com.dbms.georgia_express.model;
+
+import jakarta.persistence.*;
+
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "card")
+public class Card {
+
+    @Column(name = "card_number")
```

Comment:
This should be an @Id property

Reply:
Also you will have to modify the database to match the column name so add a corresponding flyway changes